src/item_operations.py

```diff
@@ -7,8 +7,7 @@
 
 from . import plotting_tools, graphs, utilities
 from .data import Data
-from .utilities import InteractionMode
```

Comment:
This seems to be mostly the swich from props.active_window to main_window. I think it's more robust this way, so good change.